lib/archethic/mining/validation_context.ex

```diff
@@ -1134,10 +1136,11 @@ defmodule Archethic.Mining.ValidationContext do
         fee: fee,
         transaction_movements: Transaction.get_movements(tx)
       }
-      |> LedgerOperations.from_transaction(tx)
+      |> LedgerOperations.from_transaction(tx, timestamp |> DateTime.truncate(:millisecond))
```

Comment:
The timestamp is already truncated from the mining process.
You should take the timestamp as it is.